elastic/conf.yaml.example

```diff
@@ -34,6 +34,7 @@ instances:
     # password: password
     # cluster_stats: false
     # pshard_stats: false
+    # pshard_skip_to: false  # skip if pshard stats TO
```

Comment:
is 'TO' commonly understood as 'timeout'? I had to think about this for a second before I guessed its meaning (and I only knew because of the context of this PR :))